app/__init__.py

```diff
@@ -2,15 +2,15 @@
 from flask_sqlalchemy import SQLAlchemy
 from flask_migrate import Migrate
 
-from .api import PaymentResource
 from .config import Config
 
-api = Blueprint('api', __name__)
-api.add_url_rule('/payment/', view_func=PaymentResource.as_view('payment'))
-
 app = Flask(__name__)
```

Comment:
I'm having a look, did you @matusf worked with something like this before?

Reply:
yea, I've had this issue before. @toma-s's solution is right. The issue is that browsers nowadays implement same origin security policy. (website can request only content from the same server it's being served) By adding CORS to the app we basically say browser that it's OK to make requests to other sites / servers. ([wiki](https://en.wikipedia.org/wiki/Cross-origin_resource_sharing))

things to keep in mind: 
- maybe we will need to send some token to prevent CSRF
- CORS can can be configured to allow making requests to specific addresses (if we set it to address of this API, we probably do not need CSRF token^) -> this basically means that API needs FQDN

^ the unrestricted CORS is a one-liner, but we can make it as another PR if you like

Reply:
> ^ the unrestricted CORS is a one-liner, but we can make it as another PR if you like

@matusf 
please add it in this PR
it's a blocking issue, because I cannot connect to BE

Reply:
This CR is on hold for now, as we decided to work on more pressing issues first
But the `CORS(app)` line has been merged to master a while ago ([this line](https://github.com/uk-kona/univerzita-kona-api/blob/master/app/__init__.py#L15)), so if you pull latest changes your problem should now be resolved